### ISSUE TITLE
feat: Add training visualization dashboard with TensorBoard support

### ISSUE THREAD

**Issue by newtontech** [Gen-Verse/OpenClaw-RL#14]
## Problem

当前训练过程缺少实时可视化工具，难以监控训练进度和性能指标。

### 1. 添加TensorBoard支持
```python
from torch.utils.tensorboard import SummaryWriter

class TrainingMonitor:
    def __init__(self, log_dir='./logs'):
        self.writer = SummaryWriter(log_dir)
    
    def log_metrics(self, step, metrics):
        for key, value in metrics.items():
            self.writer.add_scalar(key, value, step)
```

### 2. 关键指标监控
- Loss curves (policy_loss, value_loss)
- Reward metrics (episode_reward, average_reward)
- Training speed (samples/s, steps/s)
- GPU utilization

### 3. 可视化Dashboard
- 实时训练曲线
- 模型性能对比
- 超参数追踪

## Expected Output

- [ ] TensorBoard集成
- [ ] 关键指标日志
- [ ] 可视化教程文档

## Priority

🟢 Medium - 提升用户体验

🦞 OpenClaw AI Assistant
